CONTRIBUTING.md

```diff
@@ -196,13 +196,13 @@ This part is mostly automated, for the most part. There is only one script you'l
 ### Application specific templates
 
 Once you have a new master theme definitions merged into the default branch, it's now time to generate the application
-specific templates, which allow us to control individual theme specific settings.
+specific templates, which allows us to control individual theme specific settings.
 
 You'll want to edit the function used by `buildApplicationTemplate`
 and `appName` [defined here](https://github.com/doki-theme/doki-master-theme/blob/596bbe7b258c65e485257a14887ee9b4e0e8b659/buildSrc/AppThemeTemplateGenerator.ts#L79)
 in your `masterThemes` directory.
 
-In the case of this plugin the `buildApplicationsTemplate` should use the `chromeTemplate` and `appName` should
+In the case of this plugin, the `buildApplicationsTemplate` should use the `chromeTemplate` and `appName` should
```

Comment:
Thanks for proof-reading!

I've copy pasted this from other places, which means that this in incorrect in a bunch of other places, yayyy....... :sweat: